Hqub.MusicBrainz/Hqub.MusicBrainz.API/Extensions/Extensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Hqub.MusicBrainz.API.Extensions
+{
+    /// <summary>
+    /// Extensions to work with api
+    /// </summary>
+    /// <see href="https://musicbrainz.org/doc/Artist"/>
+    public static class HqubExtensions
+    {
+        /// <summary>
+        /// Escapes a string containing several words.
+        /// </summary>
+        /// <param name="s">Source line.</param>
+        /// <returns></returns>
+        public static string Quote(this string s)
```

Comment:
Since the extension is not connected to rest of the library, I would prefer this to be moved back to the example project.